MAUI/Rating/Appearance-Customization.md

```diff
@@ -0,0 +1,213 @@
+---
+
+layout: post
+title: Appearance Customization in .NET MAUI Rating control | Syncfusion
+description: Learn here all about Appearance Customization support in Syncfusion .NET MAUI Rating (SfRating) control and more.
+platform: maui
+control: Rating
+documentation: ug
+
+---
+
+# Appearance Customization in .NET MAUI Rating (SfRating)
+
+We can customize the rated and unrated fill color, stroke and stroke thickness using the following properties of [`RatingSettings`].
+
+* [`RatedFill`]
+* [`UnratedFill`]
+* [`RatedStroke`]
+* [`UnratedStroke`]
+* [`RatedStrokeThickness`]
+* [`UnratedStrokeThickness`]
+ 
+## Set Fill Color
+
+[`SfRating`] control has support to set the fill color for the rated and unrated items.
+
+### Rated Items
+
+The [`RatedFill`] property fills the rated item with the specified solid color in the [`SfRating`]
+
+{% tabs %}
+
+{% highlight xaml %}
+
+<rating:SfRating Value="3">
+    <rating:SfRating.RatingSettings>
+        <rating:RatingSettings RatedFill="Red"/>
+    </rating:SfRating.RatingSettings>
+</rating:SfRating>
+	
+{% endhighlight %}
+
+{% highlight C# %}
+
+SfRating rating= new SfRating();
+rating.Value = 3;
+RatingSettings ratingSettings = new RatingSettings();
+ratingSettings.RatedFill = Colors.Red;
+rating.RatingSettings = ratingSettings;
+           
+{% endhighlight %}
+
+{% endtabs %}
+
+![Rated item fill color](images/rated-fill.png)
+
+### Unrated Items
+
+The [`UnratedFill`] property fills the unrated item with the specified solid color in the [`SfRating`].
```

Comment:
The [`UnratedFill`] property fills the unrated items with the specified solid color in the [`SfRating`].

Reply:
Changed